forms/FieldList.php

```diff
@@ -412,16 +412,27 @@ public function insertAfter($name, $item) {
 	}
 
 	/**
-	 * Push a single field into this FieldList instance.
+	 * Push a single field onto the end of this FieldList instance.
 	 *
 	 * @param FormField $item The FormField to add
-	 * @param string $key An option array key (field name)
 	 */
-	public function push($item, $key = null) {
+	public function push($item) {
```

Comment:
Hm, it looks ok to me.... parent::push() doesn't have a second parameter anyway.